res/rules-engine-mqtt/configuration.toml

```diff
@@ -0,0 +1,77 @@
+[Writable]
+  LogLevel = "INFO"
+  [Writable.Pipeline]
+    ExecutionOrder = "SetOutputData"
+
+    [Writable.Pipeline.Functions.SetOutputData]
+    [Writable.Pipeline.Functions.FilterByDeviceName]
+      [Writable.Pipeline.Functions.FilterByDeviceName.Parameters]
+        DeviceNames = ""
+    [Writable.Pipeline.Functions.FilterByValueDescriptor]
+      [Writable.Pipeline.Functions.FilterByValueDescriptor.Parameters]
+        ValueDescriptors = ""
+
+[Service]
+BootTimeout = "30s"
+ClientMonitor = "15s"
+CheckInterval = "10s"
+Host = "localhost"
+Port = 50001
+Protocol = "http"
+ReadMaxLimit = 100
+StartupMsg = "AppServiceConfigurable-rules-engine has Started"
+Timeout = "5s"
+
+[Registry]
+Host = "localhost"
+Port = 8500
+Type = "consul"
+
+[Clients]
+  [Clients.CoreData]
+  Protocol = "http"
+  Host = "localhost"
+  Port = 48080
+
+  [Clients.Logging]
+  Protocol = "http"
+  Host = "localhost"
+  Port = 48061
+
+[Binding]
+Type="messagebus"
+SubscribeTopic="events"
+PublishTopic="rules-events"
+
+[MessageBus]
+Type = "mqtt"
+    [MessageBus.SubscribeHost]
+        Host = "localhost"
+        Port = 1883
+        Protocol = "tcp"
+    [MessageBus.PublishHost]
+        Host = "localhost"
+        Port = 1883
+        Protocol = "tcp"
+    [MessageBus.Optional]
+        # MQTT Specific options
+        # Client Identifiers
+        Username =""
+        Password =""
+        ClientId ="AppServiceSample"
```

Comment:
`AppServiceConfigurable`?

Reply:
Good catch meant for it to be AppServiceRulesEngine but forgot to change it...

Reply:
Fixed